setup.py

```diff
@@ -51,8 +51,8 @@
     },
 
     install_requires=[
-        'pylint==1.4.5',
-        'pylint-django==0.6.1',
+        'pylint>=1.5.4,<2.0.0',
```

Comment:
Pinning pylint here makes this library fragile. I'd rather consuming applications, especially edx-platform, pin pylint instead.